File: src/main/java/com/senderman/stickersorterbot/repository/UserRepository.java
package com.senderman.stickersorterbot.repository;

import com.senderman.stickersorterbot.model.User;
import org.springframework.data.mongodb.repository.MongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepository extends MongoRepository<User, Integer> {
}

File: src/main/kotlin/com/senderman/stickersorterbot/model/UserRepository.kt
package com.senderman.stickersorterbot.model

import org.springframework.data.mongodb.repository.MongoRepository

/**
 * To change database type for the whole application,
 * Include proper spring data dependency build.gradle.kts
 * And inherit this interface from proper interface. That's all =)
 */
interface UserRepository : MongoRepository<User, Int>